documentation/2.0/content/rcuinfo.md

```diff
@@ -64,9 +64,26 @@ domainInfo:
       oracle.net.tns_admin: <absolute path of the unzipped wallet root directory>
 
 ```
+
+For an Oracle SSL database with ONE_WAY SSL enabled, with an `SSO` wallet, use the following example:
+```yaml
+domainInfo:
+    RCUDbInfo:
+      useSSL : true
+      rcu_db_conn_string: <reuired URL string for use with -run_rcu>
```

Comment:
*required - this is spelled "reuired" in 4 places, including old and new examples

Reply:
updated, thanks for the review